nupic/research/frameworks/pytorch/imagenet/imagenet_tune.py

```diff
@@ -393,6 +410,31 @@ def _process_result(self, result, pre_experiment_result=None):
                     pprint(result)
 
 
+class DebugTrainable(Trainable):
```

Comment:
You should remove all debugging code

Reply:
I am keeping this class for future debugging, it is rather useful

Reply:
That's fair. I don't have a better name. It for debugging after all. If anything, maybe we should move it to another file in the future. Maybe add a TODO for this.